docs/source/index.rst

```diff
@@ -1,3 +1,5 @@
+:github_url: https://github.com/pytorch/vision
```

Comment:
do we need this?

Reply:
This enables edit on GitHub in docs. Not one of the best features though, but it is used in [pytorch docs](https://pytorch.org/docs/stable/index.html).
Hence I thought let's follow the same for torchvision.

![image](https://user-images.githubusercontent.com/47158509/119116640-0f2cbf80-ba46-11eb-9feb-6b5c1ec862c5.png)

Reply:
Hm, the icon for the link looks broken... let's remove it and wait until they fix it. It's not a super useful ffeature as you noted anyway

Reply:
Any idea Where could we possibly fix the icon? It is broken even on pytorch website

Reply:
The code for it is

```html
<a href="https://github.com/pytorch/pytorch" class="fa fa-github"> Edit on GitHub</a>
```

so it seems related to font-awesome somehow https://fontawesome.com/icons/github.
Perhaps the pages are missing a js script or some css or something... my web dev skills are very weak :)

Reply:
Hmm, after a little digging i found out that the pytorch_sphinx_theme creates this issue particularly [this](https://github.com/pytorch/pytorch_sphinx_theme/blob/master/pytorch_sphinx_theme/breadcrumbs.html) file

Maybe the fix is to use 

```
<i class="fa fa-github" aria-hidden="true"></i>
```

instead of `class="fa fa-github"`.

Not too sure with this as my web dev skills too are somewhat rusty.